Build failure on freebsd

**Describe the bug**
```
[2021-10-08 21:12:41.362849] [freebsd-x86_64] # github.com/shirou/gopsutil/mem
[2021-10-08 21:12:41.363248] [freebsd-x86_64] ../gopath/pkg/mod/github.com/fancybits/gopsutil@v2.16.11-0.20211008220520-cb1567181aab+incompatible/mem/mem_bsd.go:31:42: cannot use swapCommandPath (type string) as type context.Context in argument to invoke.CommandWithContext:
[2021-10-08 21:12:41.363294] [freebsd-x86_64] 	string does not implement context.Context (missing Deadline method)
```

**Environment (please complete the following information):**
 - [ ] Windows: [paste the result of `ver`]
 - [ ] Linux: [paste contents of `/etc/os-release` and the result of `uname -a`]
 - [ ] Mac OS: [paste the result of `sw_vers` and `uname -a`
 - [x] FreeBSD: [paste the result of `freebsd-version -k -r -u` and  `uname -a`]
 - [ ] OpenBSD: [paste the result of `uname -a`]

**Additional context**
[Cross-compiling? Paste the command you are using to cross-compile and the result of the corresponding `go env`]

--- a/mem/mem_bsd.go
+++ b/mem/mem_bsd.go
@@ -28,7 +28,7 @@ func SwapDevicesWithContext(ctx context.Context) ([]*SwapDevice, error) {
 	if err != nil {
 		return nil, fmt.Errorf("could not find command %q: %w", swapCommand, err)
 	}
-	output, err := invoke.CommandWithContext(swapCommandPath, "-lk")
+	output, err := invoke.CommandWithContext(ctx, swapCommandPath, "-lk")
 	if err != nil {
 		return nil, fmt.Errorf("could not execute %q: %w", swapCommand, err)
 	}

--- a/v3/mem/mem_bsd.go
+++ b/v3/mem/mem_bsd.go
@@ -28,7 +28,7 @@ func SwapDevicesWithContext(ctx context.Context) ([]*SwapDevice, error) {
 	if err != nil {
 		return nil, fmt.Errorf("could not find command %q: %w", swapCommand, err)
 	}
-	output, err := invoke.CommandWithContext(swapCommandPath, "-lk")
+	output, err := invoke.CommandWithContext(ctx, swapCommandPath, "-lk")
 	if err != nil {
 		return nil, fmt.Errorf("could not execute %q: %w", swapCommand, err)
 	}

